src/homework-03/operations.js

```diff
@@ -0,0 +1,251 @@
+let maxOffset;
+const noDisplayClass = "no_display";
+
+const activePokemons = {
+  pokemons: [],
+  domElement: "active_pokemons",
+  callback: pokemon => {
+    function fun() {
+      return movePokemonToAnotherList(pokemon, activePokemons, deletedPokemons);
+    }
+    return fun;
+  },
+  moveButtonText: "Delete"
+};
+
+const deletedPokemons = {
+  pokemons: [],
+  domElement: "deleted_pokemons",
+  callback: pokemon => {
+    function fun() {
+      return movePokemonToAnotherList(pokemon, deletedPokemons, activePokemons);
+    }
+    return fun;
+  },
+  moveButtonText: "Restore"
+};
+
+const deletedList = document.getElementById("deleted_pokemons");
+
+function fetchPokemons(offsetParam) {
+  let url = "https://pokeapi.co/api/v2/pokemon?limit=20";
+  if (offsetParam > 0) {
+    url = url + "&offset=" + offsetParam * 20;
+  }
+  console.log("Staring fetching pokemons");
+
+  fetch(url)
+    .then(response => {
+      console.log("Response", { response });
+
+      if (response.status === 200) {
+        response.json().then(jsonData => {
+          console.log("Data received", jsonData);
+          maxOffset = Math.round(jsonData.count / 20);
+          pokemons = jsonData.results;
+          console.log(this.result);
+
+          //https://stackoverflow.com/questions/3955229/remove-all-child-elements-of-a-dom-node-in-javascript
+          const domElement = document.getElementById(activePokemons.domElement);
+          domElement.textContent = "";
+          for (let i = 0; i < pokemons.length; i++) {
```

Comment:
forEach ti odmah vrati pokemona, a ne index pa je malo elegantniji
Druga opcija je [for..in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in) koju ne proporucam zbog svojih svih mogucih pitfalla